OrbitCore/LinuxPerfRingBuffer.h

```diff
@@ -36,12 +37,13 @@ class LinuxPerfRingBuffer
         // If the sizes are not the same, the memory layout defined in LinuxPerfEvent.h 
         // does not match the one found in the ring buffer.
         assert(
-            sizeof(record.ring_buffer_data) == a_Header.size && 
-            "Incorrect layout of the perf ring buffer data.\nPlease file a bug at https://github.com/pierricgimmig/orbitprofiler.\n"
+            sizeof(record.ring_buffer_data) == a_Header.size &&
+            "Incorrect layout of the perf ring buffer data.\n"
+            "Please file a bug at https://github.com/pierricgimmig/orbitprofiler.\n"
         );
     
         // Copy the data from the ringbuffer into the placeholer in that class.
-        uint8_t* dest = reinterpret_cast<uint8_t*>(&record.ring_buffer_data);
+        auto* dest = reinterpret_cast<uint8_t*>(&record.ring_buffer_data);
```

Comment:
Why change from uint8_t to auto here?

Reply:
Clang-Tidy: use auto when initializing with a cast to avoid duplicating the type name.